.github/workflows/python-cross-platform-support.yml

```diff
@@ -88,27 +88,12 @@ jobs:
           cd (Get-ChildItem -Filter aac_secure_install_*)
           .\install_aac_air_gap.bat
 
-      - name: Get Package Version
-        run: python -m aac version
-
-      - name: Check AaC DSL
-        if: matrix.os != 'windows-latest'
-        run: |
-          spec_file=$(find $pythonLocation -name aac.aac | head -n1)
-          python -m aac check $spec_file
-
-      - name: Check AaC DSL
-        if: matrix.os == 'windows-latest'
-        run: |
-          $spec_file=$(gci -r -fi aac.aac -Name "$Env:pythonLocation")
-          python -m aac check "$Env:pythonLocation\$spec_file"
-
-      - name: Print out Core Spec
-        run: python -m aac print-defs
+      - name: Test Plugin Commands
+        run: echo "Replace with testing the package commands of this repo"
```

Comment:
So, we know what these test steps need to look like for the AaC core repo still sorting it out for the stand alone plugin repos. It would be the test steps that had existed previously. So, they would need added in for this repository's pipeline.

Reply:
That would mean this generic test reminder can go away